base/doc/ltnews38.tex

```diff
@@ -207,8 +207,18 @@ \section{Extended cross-referencing of document properties}
 
 \section{New or improved commands}
 
+\section{Testing for \pkg{expl3} version: \cs{IfExplAtLeastTF}}
+
+The integration of \pkg{expl3} into the kernel means that programmers can use
+all of the features available without needing to load explicitly. However, as
+\pkg{expl3} is not a separate package, the version can't be tested using
+\cs{IfPackageAtLeastTF}. To date, low-level methods have therefore been needed
+to check for the availablity of features in \pkg{expl3}. We have now added
+\cs{IfExplAtLeastTF} as a test working in the same way as
+\cs{IfPackageAtLeastTF} but focussed on the pre-loaded programming layer.
```

Comment:
one could perhaps mention that this date shows up when LaTeX starts and also in the .log file near the end as in

LaTeX2e <2023-06-01> patch level 1
L3 programming layer <2023-10-10>

might help people to find out which date they need.